package methods_of_webdrivers;

import org.openqa.selenium.Dimension;
import org.openqa.selenium.Point;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class UseOfManage {

	public static void main(String[] args) throws InterruptedException {
WebDriver driver=new ChromeDriver(); Thread.sleep(2000);
driver.manage().window().maximize();Thread.sleep(2000);
driver.get("https://qspiders.com/");Thread.sleep(2000);
driver.manage().deleteAllCookies();Thread.sleep(2000);
driver.manage().window().setSize(new Dimension(1080, 720));Thread.sleep(2000);
driver.manage().window().setPosition(new Point(45, 18));Thread.sleep(2000);
driver.manage().window().fullscreen();Thread.sleep(2000);
driver.manage().window().minimize();Thread.sleep(2000);
driver.quit();
	}

}
